traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -214,43 +218,11 @@ INSERT INTO server (
 	:status_id,
 	:tcp_port,
 	:server_type_id,
-	:upd_pending,
 	:xmpp_id,
 	:xmpp_passwd,
 	:status_last_updated
 ) RETURNING
-	(SELECT name FROM cachegroup WHERE cachegroup.id=server.cachegroup) AS cachegroup,
-	cachegroup AS cachegroup_id,
-	cdn_id,
-	(SELECT name FROM cdn WHERE cdn.id=server.cdn_id) AS cdn_name,
-	domain_name,
-	guid,
-	host_name,
-	https_port,
-	id,
-	ilo_ip_address,
-	ilo_ip_gateway,
-	ilo_ip_netmask,
-	ilo_password,
-	ilo_username,
-	last_updated,
-	mgmt_ip_address,
-	mgmt_ip_gateway,
-	mgmt_ip_netmask,
-	offline_reason,
-	(SELECT name FROM phys_location WHERE phys_location.id=server.phys_location) AS phys_location,
-	phys_location AS phys_location_id,
-	profile AS profile_id,
-	(SELECT description FROM profile WHERE profile.id=server.profile) AS profile_desc,
-	(SELECT name FROM profile WHERE profile.id=server.profile) AS profile,
-	rack,
-	reval_pending,
-	(SELECT name FROM status WHERE status.id=server.status) AS status,
-	status AS status_id,
-	tcp_port,
-	(SELECT name FROM type WHERE type.id=server.type) AS server_type,
-	type AS server_type_id,
-	upd_pending
+	id
```

Comment:
I think you could probably leave the return values in there, although there was at least _some_ method to my madness. I originally wrote this to place the new values in a separate table. This meant inserting into the server table, getting the return values and parsing those into a server object. Then, later, one must insert the interfaces into a separate table and combine those results with the server object. And, at the time, another round inserting the server config timestamps and finally adding those to the server object. I think it's conceptually easier to perform the three inserts and one select, rather than 3 insert/selects. That all changed when I moved the columns into the `server` table, but I think this is still simpler. I could revisit this approach and revert this change now that the columns are in the `server` table.

As a side not, subqueries are often considered less performant vs joins. I did **not** run explain on the queries, so I cannot confim that this is objective improvement. Anyways, rather than performing multiple subqueries in the return call, we can now do a single select to get all the server information we need at once using joins instead of subqueries.

Reply:
> I could revisit this approach and revert this change now that the columns are in the `server` table.

@TaylorCFrey This change causes more conflicts than may be necessary with @rimashah25's #6569 PR. AFAIK #6569 plans to add them back now

Reply:
True, we should've known that these projects would collide and to minimize potential conflicts -- oops!